tensilelite/Tensile/ClientWriter.py

```diff
@@ -525,11 +526,12 @@ def param(key, value):
         param("num-benchmarks",           globalParameters["NumBenchmarks"])
         param("num-elements-to-validate", globalParameters["NumElementsToValidate"])
         param("num-enqueues-per-sync",    globalParameters["EnqueuesPerSync"])
+        param("max-enqueues-per-sync",    globalParameters["MaxEnqueuesPerSync"])
         param("num-syncs-per-benchmark",  globalParameters["SyncsPerBenchmark"])
         param("use-gpu-timer",            globalParameters["KernelTime"])
         param("hardware-monitor",         globalParameters["HardwareMonitor"])
-        if not globalParameters["KernelTime"]:
-            param("num-warmups", 1)
+        param("num-warmups",              globalParameters["NumWarmups"])
```

Comment:
No default "NumWarmups", if config yaml didn't set it , it would cause python error